plugins/org.eclipse.n4js.smith/src/org/eclipse/n4js/smith/TimedMeasurement.java

```diff
@@ -35,7 +35,7 @@ synchronized long elapsed(TimeUnit timeUnit) {
 	}
 
 	@Override
-	synchronized public void end() {
```

Comment:
`synchronized` was necessary before to avoid multithread problems. are you sure you can just remove it?

Reply:
good point! didn't remove that on purpose